_projects/implement_metrics_for_coala.md

```diff
@@ -1,7 +1,8 @@
 ---
 collaborating_projects:
   - coala
-desc: "There is more to software quality than just passing builds - coala should support generating metrics for your code."
+desc: "There is more to software quality than just passing builds - coala should support generating metrics for
```

Comment:
break this line at 80cpl, and align indented text on next line under first letter of this line.